spec/cucumber/glue/proto_world_spec.rb

```diff
@@ -68,35 +68,39 @@ module Glue
 
           define_steps do
             When('an object is logged') do
-              log(a: 1, b: 2, c: 3)
+              object = Object.new
+              def object.to_s
+                '<test-object>'
+              end
+              log(object)
             end
           end
 
-          it 'attached the styring version on the object' do
-            expect(@out.string).to include '{:a=>1, :b=>2, :c=>3}'
+          it 'attached the string version on the object' do
+            expected_string = '<test-object>'
+            expect(@out.string).to include expected_string
           end
         end
 
         describe 'when logging multiple items on one call' do
           define_feature <<-FEATURE
-        Feature: Banana party
+        Feature: Logging multiple entries
 
-          Scenario: Monkey eats banana
-            When monkey eats banana
+          Scenario: Logging multiple entries
+            When logging multiple entries
           FEATURE
 
           define_steps do
-            When('{word} {word} {word}') do |subject, verb, complement|
-              log "subject: #{subject}", "verb: #{verb}", "complement: #{complement}", subject: subject, verb: verb, complement: complement
+            When('logging multiple entries') do
+              log 'entry one', 'entry two', 'entry three'
             end
           end
 
-          it 'logs each parameter independently' do
+          it 'logs each entry independently' do
```

Comment:
This feels super weird to me. I'll be raising this in the contributor meeting as I'm curious why we ever built it to do this. But yes this is correct.

Can you wrap the include in `()`